docs/benchmark.md

```diff
@@ -60,13 +61,13 @@ This will
 - Run the benchmark tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to Unix: `unix:///var/run/dockershim.sock` or Windows: `tcp://localhost:3735` by default. For other runtimes, the endpoint can be set by flags `--runtime-endpoint` and `--image-endpoint`.
+critest connects to Unix: `unix:///var/run/cri-dockerd.sock` or Windows: `npipe:////./pipe/cri-dockerd` by default. For other runtimes, the endpoint can be set by flags `--runtime-endpoint` and `--image-endpoint`.
```

Comment:
Is this statement true? Defaults are "unix:///run/containerd/containerd.sock", "unix:///run/crio/crio.sock", "unix:///var/run/cri-dockerd.sock"

Reply:
Those defaults are only for `crictl`, while this documentation was for `critest`.

The defaults are deprecated anyway, so both crictl and critest _should_ be configured...

Reply:
There was no change in behavior, it was running cri-dockerd also before - it just moved the endpoint location to use the old dockershim path...

Removing the defaults could definitely be done.